Configuration/TypoScript/Plugins/media.typoscript

```diff
@@ -9,5 +9,28 @@ plugin.tx_slubwebsachsendigital_media {
     playerTranslations {
       baseFile=EXT:slub_web_sachsendigital/Resources/Private/Language/locallang_video.xlf
     }
+
+    shareButtons {
+      0 {
+        type = material
+        icon = email
+        titleTranslationKey = share.email.tooltip
+        hrefTemplate = mailto:?body={url}%0A%0A
+      }
+
+      1 {
+        type = material
+        icon = qr_code
+        titleTranslationKey = share.qr_code.tooltip
+        hrefTemplate = dlf:qr_code
+      }
+
+      2 {
+        type = image
+        src = /fileadmin/user_upload/Logo_blue.svg
```

Comment:
I would suggest to add the logo to `Resources/Public/Images` or similar. Than this will work out-of-the box. For customization, the integrator may overwrite the TypoScript.

Reply:
Thank you!